components/widgets/profile-info.js

```diff
@@ -1,7 +1,75 @@
 import { UserOutlined } from "@ant-design/icons";
+import { useEffect } from "react";
+import { fetchTiktokProfile } from "@/services/viralhub-api";
 
-const ProfileInfo = () => {
-  return <div>Demo Chart 1</div>;
+const ProfileInfo = (props) => {
+
+  // This for testing if frontend can access backend
+  // useEffect(() => {
+  //   console.log("UseEffect")
+  //   const response = fetchTiktokProfile(['gordonramsayofficial'])
+  //   console.log(response);
+  // }, []);
+
+  return(
+    <>
+      <div className="grid grid-cols-3 gap-4 sm:grid sm:grid-cols-3 sm:gap-4">
+        <div className="bg-blue-200 p-4">
+          <div className="inline-flex flex-row">
+            <h1 className="ml-3">Profile: {props.nickName}</h1>
+          </div>
+          <div className="mt-3">
+            <img 
+              src="https://hips.hearstapps.com/hmg-prod/images/gordon-ramsay-646367718a5f4.jpg?crop=1xw:1xh;center,top&resize=980:*"
```

Comment:
Will need to be changed to the props.avatar provided URL. This url point to the cdn where tiktok hosts the image at.